lib/control/v1/authenticate.js

```diff
@@ -18,6 +19,9 @@ function Authenticate(app, server, vtoken) {
   app.post('/v1/authenticate',
     (req, res, next) => {
       console.log('authenticate request recieved');
+      if (!Config.get('vault:unseal')){
+        return bad.request(res, 'Warden not unsealed');
```

Comment:
Should this error message be "Vault is sealed! Make sure the 'vault:unseal' configuration setting is 'true'."? That way people getting the error have an idea of how to fix it.